tests/integration/test_filtering.py

```diff
@@ -8,9 +8,12 @@
 # Avoid assertions using equality to prevent intermittent failures due to
 # other builds simultaneously adding projects to test org
 def test_where(client, image_url, rand_gen):
-    p_a = client.create_project(name="a", queue_mode=QueueMode.Batch)
```

Comment:
This test is flaky and I have been trying to fix it... if it works I could pull it into a separate PR or leave it... currently it did not work anyway

Reply:
The tests are green so perhaps I have fixed it. May I suggest leaving it as a part of this PR as the time of making and merging new PR for a small change like this might be high.